### PR TITLE
[DCOS-55014] add drain status to node details

### PR DESCRIPTION
**Pull request by brandonc**
## Testing

There is a bug that prevents the mesos state store from loading when useFixtures is true (this does not seem to affect integration tests?) Let me know if you are affected by the same issue. If so, just bypass the state store loading by changing NodeDetailPage line 41 from `mesosStateLoaded: false` to `mesosStateLoaded: true`

Visit the node details page (details tab) and see the drain status and new section there

## Dependencies

This PR is currently based on #3960 

## Screenshots

Screen Shot 2019-07-01 at 3 01 25 PM

### CODE REVIEW

**Comment by GeorgiSTodorov** [comment]
Build is failing because of the commit message.

File: plugins/nodes/src/js/pages/nodes/NodeDetailTab.js [hunk 1]
```diff
@@ -57,12 +59,24 @@ class NodeDetailTab extends PureComponent {
...
   }
 
+  hasMaintenanceData() {
+    const { node } = this.props;
+
+    // Detecting whether DCOS already supports maintenance mode.
+    // We might want to remove this flag at some point in the future.
+    return (
+      typeof node.get("drain_info") !== "undefined" ||
+      typeof node.get("deactivated") !== "undefined"
```
Comment (GeorgiSTodorov) [inline plugins/nodes/src/js/pages/nodes/NodeDetailTab.js]: Why are we checking the type and not just the value?

Comment (pierrebeitz) [reply plugins/nodes/src/js/pages/nodes/NodeDetailTab.js]: that's a really good question. will change in #3960 as well!

**Comment by GeorgiSTodorov** [comment]
> Let me know if you are affected by the same issue.

I am.

Nitpick: I think it looks better with a slightly smaller margin-left:
Original 7px margin.
Screenshot from 2019-07-02 10-16-31

4px margin.
Screenshot from 2019-07-02 10-16-51

**Comment by GeorgiSTodorov** [comment]
It's failing again. I think it's because it's not pointing to master.

**Comment by pierrebeitz** [comment]
> Nitpick: I think it looks better with a slightly smaller margin-left: 

i used the same paddings as are used in the services table in case someone wants to clean that up at some point. `7px` is pretty nice to grep for.

**Comment by brandonc** [comment]
Going ahead and merging on the basis of previous approvals

**Pull request merged**